### PR TITLE
[depends] - build fixes for libxslt and openssl

### PR DESCRIPTION
**Pull request by shadamik**

### CODE REVIEW

**Comment by wsnipex** [comment]
@Memphiz ping

**Comment by wsnipex** [comment]
jenkins build this please

**Comment by Memphiz** [comment]
Its fine beside that sysroot change which should use $(SDKROOT) instead pretty sure.

**Comment by shadamik** [comment]
The sysroot is already injected by CFLAGS, which is the common way for cross set ups. Thats why I removed the uncommon way that openssl uses. 

Why pass  sysroot twice?

File: tools/depends/target/openssl/Makefile [hunk 1]
```diff
@@ -13,9 +13,8 @@ ifeq ($(OS), android)
   CONFIGURE=./Configure shared zlib --openssldir=$(PREFIX) --with-zlib-include=$(PREFIX)/include --with-zlib-lib=$(PREFIX)/lib linux-generic32
 endif
 ifeq ($(OS), ios)
-  # No darwin-arm-cc so use darwin-i386-cc and patch files after configure
-  CONFIGURE=./Configure darwin-i386-cc zlib no-asm no-krb5 --openssldir=$(PREFIX)
-endif
+  CONFIGURE=./Configure iphoneos-cross zlib no-asm no-krb5 --openssldir=$(PREFIX)
+  endif
```
Comment (Rechi) [inline tools/depends/target/openssl/Makefile]: indentation

**Comment by Memphiz** [comment]
@shadamik sorry i misread. You are removing the flags i see. I just stumbled over the used variables (CROSS_TOP) for generating the sysroot path while we have the SDKROOT var already in place which points to that path. Fine by me now that i read that sed right...

**Comment by Memphiz** [comment]
??

**Comment by wsnipex** [comment]
jenkins build and merge

**Comment by shadamik** [comment]
sorry, my fault: comment string in multiline bash command. I will fix this now...

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Pull request merged**